src/components/IncomeComponent.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Paper,
+  Typography,
+} from '@material-ui/core';
+import Loading from './LoadingComponent';
+import IncomeGrossPay from '../container/IncomeGrossPayContainer';
+import IncomeDeductions from '../container/IncomeDeductionsContainer';
+
+class IncomeComponent extends Component {
+  componentDidMount = async () => {
+    const {
+      doLoadIncome,
+    } = this.props;
+    await doLoadIncome();
+  }
+
+  render = () => {
+    const {
+      isLoading,
+      currency,
+      netPay,
+    } = this.props;
```

Comment:
Wieso machst du jeweils eine Variable am Anfang im render() mit allen props?
Du könntest auch gleich direkt abfragen this.props.isLoading.
Fände ich persönlich schöner oder macht man das so? :)